.gitignore

```diff
@@ -24,6 +24,13 @@ automappingconverter.exe
 *.lib
 *.exp
 *.intermediate.manifest
+src/libtiled/debug/
+src/tiledquick/debug/
+src/tiledquick/uic_wrapper.bat
+src/tiledquickplugin/debug/
```

Comment:
Why the `debug/` directory only for these three projects? And why `uic_wrapper.bat` for `tiledquick`, which doesn't use `uic` at all? (it's actually the first time I see that file)